packages/gateway/src/features/fhir/FHIRAPI.ts

```diff
@@ -49,7 +49,10 @@ export default class FHIRAPI extends RESTDataSource<Context> {
       }
     }
 
-    return this.get<Practitioner>(`/Practitioner/${practitionerId}`)
+    const res = await this.get<Practitioner>(
+      `${fhirPath}/Practitioner/${practitionerId}`
+    )
+    return res
```

Comment:
Something related to this had changed in Apollo but yea, it might not be that we have to do it
https://github.com/apollographql/datasource-rest#baseurl
There are some documentation here. I'll play around with it a bit. I think it might have to be so that the baseUrl path needs to end in a slash

>If the base URL has a path element and does not end in a slash, then the given path replaces the last element of the path. That is, if baseURL is https://foo.com/a/b/c, this.get('d') resolves to https://foo.com/a/b/d

Reply:
`http://localhost:3447/fhir` is our default value currently it seems. I wouldn't want to change the configuration soo.. I can do `baseUrl = FHIR_URL.endsWith('/') ? FHIR_URL : FHIR_URL + '/'`